src/baseline_bots/bots/smart_order_accepter_bot.py

```diff
@@ -528,27 +530,25 @@ def __call__(self, rcvd_messages: List[Tuple[int, Message]]):
         yield self.replace_aggressive_order_to_allies()
 
         # generate messages for FCT sharing info orders
+        opps = list(powers.keys()).copy()
+        opps.remove(self.power_name) # list of opposing powers
         msgs_data = self.gen_messages(orders_data.get_list_of_orders())
+        if self.game.phase == "SPRING 1901 MOVEMENT":
+            for pow in opps:
+                msgs_data.add_message(pow, f"ALY ({self.power_name} {pow}) VSS ({[country for country in list(powers.copy().keys()) if country != pow and country != self.power_name]})")
```

Comment:
Use `" ".join([country for country in list(powers.copy().keys()) if country != pow and country != self.power_name])`